R/lookup_tables.R

```diff
@@ -148,9 +149,10 @@ lookup_labels <- c(
   , "parameter.num.df"   = "$\\mathit{df}_1$"
   , "parameter.denom.df" = "$\\mathit{df}_2$"
   # p.value
-  , "p.value"    = "$p$"
-  , "Pr..Chisq." = "$p$"
-  , "Pr..F."     = "$p$"
-  , "Pr..PB."    = "$p$"
-  , "pvalues"    = "$p$"
+  , "p.value"     = "$p$"
+  , "Pr..Chisq."  = "$p$"
+  , "Pr..F."      = "$p$"
+  , "Pr..PB."     = "$p$"
+  , "pvalues"     = "$p$"
+  , "adj.p.value" = "$p_{adj}$"
```

Comment:
The obsessive-compulsive me wants `"$p_\\mathit{adj}$"` or `"$p_\\mathrm{adj}$"` here. Just imagine the weird spacing! :laughing: